File: DailyFlash Solutions/DailyFlash Solutions/31-DailyFlash_Solutions/18_Feb_Solution_Three/C++/prog5.cpp
#include<iostream>

int main() {
	int num;
	std::cout << "Enter num : ";
	std::cin >> num;
	int temp = num;
	std::cout << "Perfect divisors : ";
	while(num != 0) {
		if(temp % (num % 10) == 0) {
			std::cout << num % 10 << " ";
		}
		num /= 10;
	}
	return 0;
}

File: DailyFlash Solutions/DailyFlash Solutions/31-DailyFlash_Solutions/18_Feb_Solution_Three/Java/prog5.java
import java.util.Scanner;

class Demo {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter num : ");
		int num = sc.nextInt();
		int temp = num;
		System.out.print("Divisors : ");
		while(num != 0) {
			if(temp % (num % 10) == 0) {
				System.out.print(num % 10 + " ");
			}
			num /= 10;
		}

	}
}
